pkg/pipelines/deployment/deployment.go

```diff
@@ -91,10 +95,12 @@ func ptr32(i int32) *int32 {
 	return &i
 }
 
-func labelSelector(name, value string) *metav1.LabelSelector {
+//LabelSelector used to create the labelSelector for the commit status tracker
+func LabelSelector(name, partOf string) *metav1.LabelSelector {
 	return &metav1.LabelSelector{
 		MatchLabels: map[string]string{
-			name: value,
+			KubernetesAppNameLabel: name,
+			KubernetesPartOfLabel:  partOf,
```

Comment:
I don't think we want to select on "partOf" label.   You need to double check.

Reply:
So we decided to keep both the labels